src/index.html

```diff
@@ -11,7 +11,195 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div>
+          <label for="surname">
+            Surname:
+            <input
+              id="surname"
+              type="text"
+              name="surname"
+              maxlength="20"
+            >
+          </label>
+        </div>
+        <br>
+        <div>
+          <label for="name">
+            Name:
+            <input
+              id="name"
+              type="text"
+              name="name"
+              maxlength="20"
+            >
+          </label>
+        </div>
+        <br>
+        <div>
```

Comment:
add one empty line between same level elements
```suggestion
        </div>
        
        <div>
```